library/include/hipsparse.h

```diff
@@ -38,6 +38,8 @@
 #ifndef HIPSPARSE_H
 #define HIPSPARSE_H
 
+#define INSTANTIATE_TEST_SUITE_P INSTANTIATE_TEST_CASE_P
```

Comment:
Why this?

Reply:
Accidently left in when I was running on system with old gtest version. Removed it now.